quotefault/__init__.py

```diff
@@ -85,13 +108,13 @@ def get():
     quote_lst_old = [] #for quotes older than a month, reduces clutter by hiding these behind a collapsable section
     quoteCount = 0
     for quote_obj in reversed(quotes):
-        if(quoteCount < 30):
+        if quoteCount < 30:
             quote_lst_new.append(
-                " \"" + quote_obj.quote + "\" - " + quote_obj.speaker + ", submitted by " + quote_obj.submitter + " on " + str(
-                    quote_obj.quoteTime))
+                " \"" + quote_obj.quote + "\" - " + quote_obj.speaker + ", said on " + str(quote_obj.quote_date) + ", submitted by " + quote_obj.submitter + " on " + str(

```

Comment:
Instead of duplicating this code with `randomQuote`, move this code into a method that knows how to format a quote. Ideally, put it in the `__str__` method of `Quote` itself, so you can just do `str(quote_obj)`.